node/core/av-store/src/lib.rs

```diff
@@ -315,6 +318,7 @@ impl PartialOrd for ChunkPruningRecord {
 pub struct AvailabilityStoreSubsystem {
 	pruning_config: PruningConfig,
 	inner: Arc<dyn KeyValueDB>,
+	chunks_cache: LruCache<(CandidateHash, u32), ErasureChunk>,
```

Comment:
Why is that not per candidate?

Reply:
However I think a lru cache is a bad idea.

We should keep the chunks as long as a candidate is seen as active and not included or time out. 

The random size of 32 chunks will fail in the moment we increase the number of parachains/validators.

Reply:
done. Cache will be cleared on either candidate inclusion or timeout together with pruning.